### PR TITLE
Initial GitHub Actions config

### PR DESCRIPTION
**Pull request by morancj**
Largely lifted from https://docusaurus.io/docs/deployment#triggering-deployment-with-github-actions

### CODE REVIEW

File: .github/workflows/documentation.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+name: documentation
+
+on:
+  pull_request:
+    branches: [documentation]
```
Comment (morancj) [inline .github/workflows/documentation.yml:5]:
```suggestion
    branches: [gh-pages]
```
?

File: .github/workflows/documentation.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  push:
+    branches: [documentation]
```
Comment (morancj) [inline .github/workflows/documentation.yml:7]:
```suggestion
    branches: [gh-pages]
```
?

File: .github/workflows/documentation.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+          npm run build
+  gh-release:
+    if: github.event_name != 'pull_request'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@50fbc62
+      - uses: actions/setup-node@56899e0
+        with:
+          node-version: '12.x'
+      - uses: weaveworks/webfactory-ssh-agent@6b2f2c5
```
Comment (morancj) [inline .github/workflows/documentation.yml:36]: Forked from upstream.

File: .github/workflows/documentation.yml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+          npm run build
+  gh-release:
+    if: github.event_name != 'pull_request'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@50fbc622fc4ef5163becd7fab6573eac35f8462e
+      - uses: actions/setup-node@56899e050abffc08c2b3b61f3ec6a79a9dc3223d
+        with:
+          node-version: '12.x'
+      - uses: weaveworks/webfactory-ssh-agent@6b2f2c5354ff41f1edbbf7a17ea9b6178c89be9f
```
Comment (morancj) [inline .github/workflows/documentation.yml:36]: Forked from upstream.

File: docusaurus.config.js [hunk 5]
```diff
@@ -6,7 +6,7 @@ module.exports = {
   title: 'WKP Technical Documentation',
   // tagline: 'The tagline of my site',
   url: 'https://weaveworks.github.io/wkp-docs/',
-  baseUrl: '/',
+  baseUrl: '/wkp-docs/',
```
Comment (morancj) [inline docusaurus.config.js:9]: Per https://docusaurus.io/docs/deployment#deploying-to-github-pages

Comment (luizbafilho) [reply docusaurus.config.js:9]: this is only true if we use the default github domain. Since we are using the custom domain it should remain as `/`

**Review by luizbafilho — APPROVED** [review]
LGTM but a minor change in the config

File: docusaurus.config.js (hunk 5 above)
Comment (morancj) [reply docusaurus.config.js:9]: OK: Do you have a reference for that, or just from experience? TIA

Comment (morancj) [inline docusaurus.config.js:9]:
```suggestion
  baseUrl: '/',
```

Comment (luizbafilho) [reply docusaurus.config.js]: just from experience

**Pull request merged**